__tests__/index.test.js

```diff
@@ -526,6 +526,125 @@ describe("Purchases", () => {
     expect(NativeModules.RNPurchases.checkTrialOrIntroductoryPriceEligibility).toBeCalledWith(["monthly"]);
   })
 
+  it("getPaymentDiscount works", () => {
+    const Purchases = require("../index").default;
+
+    NativeModules.RNPurchases.getPaymentDiscount.mockResolvedValue(paymentDiscountStub);
+
+    const aProduct = {
+      ...productStub,
+      discounts: [discountStub]
+    }
+
+    Purchases.getPaymentDiscount(aProduct, discountStub)
+
+    expect(NativeModules.RNPurchases.getPaymentDiscount).toBeCalledWith(aProduct.identifier, discountStub.identifier);
+    expect(NativeModules.RNPurchases.getPaymentDiscount).toBeCalledTimes(1);
+  });
+
+  it("getPaymentDiscount returns undefined for Android", async () => {
```

Comment:
minor, but the standard syntax for these is supposed to read sort of like a phrase, like when they fail:
"when getting payment discount it returns undefined for android failed"
would be expressed as 

```javascript
context("getting payment discount", () => { 
    it ("returns undefined for Android", () => {
```

see https://github.com/rubocop-hq/rspec-style-guide

Reply:
oh cool